grails-app/views/opus/_report.gsp

```diff
@@ -84,4 +87,84 @@
+        </div>
+    </div>
+
+    <div class="well top-buffer" ng-show="reportCtrl.selectedPeriod.id == 'custom'">
+        <h4>Choose start and end dates</h4>
+        <div class="row">
+            <div class="col-md-6">
+                <label class="control-label col-sm-2" for="inputFromDate">Start date:</label>
+                <div>
+                    <p class="input-group">
+                        <input type="text" id="inputFromDate" class="form-control" ng-required="true"
+                               is-open="reportCtrl.isFromOpen" datepicker-popup="dd-MMMM-yyyy" ng-model="reportCtrl.dates.from"/>
+                        <span class="input-group-btn">
+                            <button class="btn btn-default" type="button" ng-click="reportCtrl.open('from',$event)">
+                                <i class="glyphicon glyphicon-calendar"></i>
+                            </button>
+                        </span>
+                    </p>
+                </div>
+            </div>
+            <div class="col-md-6">
+                <label class="control-label col-sm-2" for="inputToDate">End date:</label>
+                <div>
+                    <p class="input-group">
+                        <input type="text" id="inputToDate" class="form-control" ng-required="true"
+                               ng-model="reportCtrl.dates.to" is-open="reportCtrl.isToOpen" datepicker-popup="dd-MMMM-yyyy"/>
+                        <span class="input-group-btn">
+                            <button class="btn btn-default" type="button" ng-click="reportCtrl.open('to', $event)">
+                                <i class="glyphicon glyphicon-calendar"></i>
+                            </button>
+                        </span>
+                    </p>
+                </div>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-md-12">
+                <button type="button" class="btn btn-primary pull-right" ng-disabled="!reportCtrl.checkFormValid()" ng-click="reportCtrl.loadCustomDateReport()">Look up</button>
+            </div>
+        </div>
+    </div>
+
+    <div class="table-responsive">
+        <table class="table table-striped" ng-show="reportCtrl.reportData.records.length > 0">
+            <thead>
+            <tr><th>Profile</th><th>Date modified</th><th>Editor</th></tr>
+            </thead>
+            <tbody>
+            <tr ng-repeat="profile in reportCtrl.reportData.records">
+                <td>
+                    <a href="${request.contextPath}/opus/{{ reportCtrl.opusId }}/profile/{{ profile.scientificName }}"
+                       target="_blank" class="scientific-name">{{profile.scientificName}}</a>
+                </td>
+                <td>
+                    {{ profile.lastUpdated | date:'dd/MM/yyyy h:mm a' }}
+                </td>
+                <td>
+                    {{ profile.editor || 'Unknown' }}
```

Comment:
You can use the angularjs default filter instead of ||
{{ profile.editor | default:'Unknown' }}